src/Mutant/MutantInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Mutant;
+
+use Infection\MutationInterface;
+
+interface MutantInterface
+{
+    public function getMutatedFilePath(): string;
```

Comment:
suggestion: IMHO it'll be a good idea to add docblock with description on each method + class name with information about what each method usually do. In the same way like symfony has. (as example https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpKernel/Controller/ControllerResolverInterface.php)

p.s. feel free to leave as is if it doesn't make sense for u.

Reply:
Yes, that's sensible. I'll add a `@see` tag, but a full description is beyond the purpose of this PR.

Reply:
OK, done.